<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralWidget</class>
 <widget class="QWidget" name="GeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>412</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>General information:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Here you can enter optional general information on the metalink.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Origin:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="origin">
       <property name="clickMessage">
        <string>Web URL to the metalink</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDynamic">
       <property name="toolTip">
        <string>Dynamic means that updated metalinks can be found at &quot;Origin&quot;.</string>
       </property>
       <property name="text">
        <string>Dynamic:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="dynamic">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="publishedGroupBox">
     <property name="title">
      <string>Metalink published</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Date and time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KDateTimeWidget" name="published"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="use_publishedtimeoffset">
        <property name="text">
         <string>Timezone offset:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="publishedNegative">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Negative offset:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="publishedoffset"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="updatedGroupBox">
     <property name="title">
      <string>Metalink updated</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="2" column="1">
       <widget class="QCheckBox" name="updatedNegative">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Negative offset:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="use_updatedtimeoffset">
        <property name="text">
         <string>Timezone offset:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KDateTimeWidget" name="updated"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Date and time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="updatedoffset"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>kdatetimewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>use_publishedtimeoffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>publishedNegative</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_updatedtimeoffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>updatedNegative</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_updatedtimeoffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_publishedtimeoffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
